lighthouse-core/test/gather/driver-test.js

```diff
@@ -138,6 +141,14 @@ describe('Browser Driver', () => {
     });
   });
 
+  it('throws if getRequestContent takes too long', () => {
+    return driverStub.getRequestContent(0, MAX_WAIT_FOR_PROTOCOL).then(_ => {
+      assert.ok(false, 'long-running getRequestContent supposed to reject');
+    }).catch(e => {
+      assert.equal(e.code, 'REQUEST_CONTENT_TIMEOUT');
```

Comment:
will the above assert message get eaten by this comparison? could also do the two function `then()`
```js
return driverStub.getRequestContent(0).then(value => {
  assert.ok(false, 'long-running getRequestContent supposed to reject');
}, e => {
  assert.equal(e.code, 'REQUEST_CONTENT_TIMEOUT');
});
```